db/config.cc

```diff
@@ -1024,6 +1024,10 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "Ignored if authentication tables already contain a super user password.")
     , auth_certificate_role_queries(this, "auth_certificate_role_queries", value_status::Used, { { { "source", "SUBJECT" }, {"query", "CN=([^,]+)" } } },
         "Regular expression used by CertificateAuthenticator to extract role name from an accepted transport authentication certificate subject info.")
+    , persistent_toppartitions_publish_interval_sec(this, "persistent_toppartitions_publish_interval", liveness::LiveUpdate, value_status::Used, 10, "Controls how often the toppartitions results will be published to the logs.")
+    , persistent_toppartitions_capacity(this, "persistent_toppartitions_capacity", liveness::LiveUpdate, value_status::Used, 256, "Controls the per shard capacity that will be used to calculate the toppartitions.")
+    , persistent_toppartitions_list_size(this, "persistent_toppartitions_list_size", liveness::LiveUpdate, value_status::Used, 20, "Controls how many partitions per shard will be showed in the toppartitions results.")
```

Comment:
Not clear what the difference is between list size and capacity. I though they are the same but apparently not. What is "capacity"?

Reply:
The `list_size` is the number of partitions that we're going to show in the results.
The `capacity` represents the maximum number of elements that we're going to track while we're calculating the toppartitions. We do this to limit the amount of space that is going to be used while sampling the toppartitions. The general idea is explained here -> https://github.com/scylladb/scylladb/issues/2811#issuecomment-420212450